build-tools/Dockerfile.debian.builder

```diff
@@ -53,6 +53,17 @@ ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
 
 RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
 WORKDIR $GOPATH
+# install gosu based on host architecture
+# https://github.com/tianon/gosu/blob/master/INSTALL.md#from-centos
+ENV GOSU_VERSION 1.10
+RUN set -ex && \
+        dpkgArch="$(dpkg --print-architecture | awk -F- '{ print $NF }')" && \
+        wget -O /usr/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch" && \
+        wget -O /usr/local/bin/gosu.asc "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch.asc" && \
+        chmod +x /usr/bin/gosu && \
+        chmod +x /usr/local/bin/gosu.asc && \
+# verify that the binary works
+        gosu nobody true
```

Comment:
Not sure how this will impact existing builds. Did you test these against different architectures?

Reply:
@amudukutore same logic has been using F5 k8s-bigip-ctlr rhel7-builder.
https://github.com/F5Networks/k8s-bigip-ctlr/blob/master/build-tools/Dockerfile.rhel7.builder#L38-#L49